src/nvim/os_unix.c

```diff
@@ -202,6 +202,13 @@ int mch_expand_wildcards(int num_pat, char_u **pat, int *num_file,
   static char *sh_vimglob_func =
     "vimglob() { while [ $# -ge 1 ]; do echo \"$1\"; shift; done }; vimglob >";
 
+  bool is_fish_shell =
+#if defined(UNIX)
```

Comment:
Fish in cygwin could be invoked from GUI on Windows, can leave out the guard. 

Or did we guard it in make_filter_cmd ?